Frontend/VIAProMa/Assets/Scripts/UI/Main Menu Cube/MainMenu.cs

```diff
@@ -32,6 +34,9 @@ public class MainMenu : MonoBehaviourPunCallbacks
         [SerializeField] private GameObject visualizationShelfPrefab;
         [SerializeField] private GameObject loadShelfPrefab;
         [SerializeField] private GameObject avatarConfiguratorPrefab;
+        [SerializeField] private AnchorManager anchoringManager;
+        [SerializeField] private Transform anchorTransform;

```

Comment:
Instead of setting them explicitly in the inspector, you could directly access the singleton by `AnchoringManager.Instance`. It can also contain the reference to the anchor object to get its transform using the manager.